src/app/components/MainMenu.js

```diff
@@ -0,0 +1,77 @@
+import createVisualImage from './VisualImage';
+import createLogo from './Logo';
+import createButtonRed from './ButtonRed';
+import createWhiteButtonWithIcon from './ButtonWhiteWithIcon';
+import createGameModeName from './GameModeName';
+import ModeMenu, { menuOption } from './ModeMenu';
```

Comment:
https://github.com/benmosher/eslint-plugin-import/blob/v2.22.1/docs/rules/no-cycle.md

Is it possible to solve it? 🔝